gemlog/parsers.pyx

```diff
@@ -31,9 +32,8 @@ def parse_gemfile(filename):
         msg = "No such file or directory: '{}'".format(filename)
         raise FileNotFoundError(2, msg)
 
-    cdef char * line = NULL
-    cdef size_t line_length = 0
-    cdef ssize_t read
+    cdef char line[256]
```

Comment:
Just to make sure, this is a reasonable line length limit right?  `getline` mallocs as big a buffer as needed, but `fgets` expects a pointer to an existing buffer.

Reply:
Yes, no line will be that long.